scripts/minikube.sh

```diff
@@ -175,7 +175,7 @@ fi
 K8S_FEATURE_GATES=${K8S_FEATURE_GATES:-"BlockVolume=true,CSIBlockVolume=true,VolumeSnapshotDataSource=true,ExpandCSIVolumes=true"}
 
 #extra-config for kube https://minikube.sigs.k8s.io/docs/reference/configuration/kubernetes/
-EXTRA_CONFIG=${EXTRA_CONFIG:-"--extra-config=apiserver.enable-admission-plugins=PodSecurityPolicy"}
+EXTRA_CONFIG_PSP="--extra-config=apiserver.enable-admission-plugins=PodSecurityPolicy"
```

Comment:
@nixpanic Will it be ideal to split the changes to minikube.sh into a separate patch ?

Reply:
The change here is really needed for the e2e testing as the `--read-only-port` option is required to be passed at the 1st (and maybe subsequent) `minikube start` commands, that conflicts with the PSP option. I do not think there is much benefit in adding it as a separate commit.

Reply:
I just asked for this as the context was not clear for me.